### ISSUE TITLE
The Assumptions Underlying the Two-Escrow Model

### ISSUE THREAD

**Issue by avnr** [Voluntarynet/Bitmarkets#32]
Would like to have your opinion about the following, I'm sure these are use cases that you considered yet they are not discussed in the whitepaper:

**1. Seller is typically better informed than Buyer about the nature of the traded goods.**

Somewhat extreme example: Parties agree to trade certain goods for X btc and lock the funds in escrow. Then Seller confesses that she is a swindler and has no goods to sell. So Seller is in a better position because her risk is only X btc whereas Buyer has 2X btc to risk. The best "amicable" solution for Buyer is to agree to split the 3X btc in two, lose 25% of his investment, and let Seller make 50% on her investment. If Buyer is more stressed then Seller on ever seeing his money again then he may actually agree on far worse terms.

The above plot can happen in a milder version under several scenarios, For example: The parties dispute whether a defect in the goods happened under the Seller’s or Buyer’s possession. Again, Seller has an advantage, because if she withheld her knowledge about the defect then she probably also factored her risk in advance.

**2. The Buyer has an advantage when buying a tradeable commodity (e.g. gold bullions)**

Once Buyer receives the goods 50% of his investment is liquid whereas Seller’s entire investment is locked. It seems that he is in a better position to re-negotiate the deal and extort better terms.

**3. The two-escrow model will fail when the identity of one party is known to the other, and the other party has non-financial motives to damage the first party (political, romantic, revenge, metal illness, etc.).**

In other words, one of the parties may lock the other's funds for non-financial reasons. The assumption that humans are reasonable isn't always true.

**4. The two-escrow model is susceptible to out-of-band events.**

One of the parties may get arrested, involved in an accident, lose their private keys, etc. those things happen, and under this model it seems that the other party remains with no remedy.

Thanks!

**Comment by stevedekorte** [comment]
Hi Avner,

Thanks for your interest and questions.

> 1. Seller is typically better informed than Buyer about the nature of the traded goods.
> 
> Somewhat extreme example: Parties agree to trade certain goods for X btc and lock the funds in escrow. Then Seller confesses that she is a swindler and has no goods to sell. So Seller is in a better position because her risk is only X btc whereas Buyer has 2X btc to risk. The best "amicable" solution for Buyer is to agree to split the 3X btc in two, lose 25% of his investment, and let Seller make 50% on her investment. If Buyer is more stressed then Seller on ever seeing his money again then he may actually agree on far worse terms.

It's true that the system relies on the participants being willing to act punitively at their own loss, which the results on the Ultimatum Game experiments suggest they are willing to do as long as the value lost isn't large compared to their monthly income. Also, the app doesn't allow for partial refunds.

> The above plot can happen in a milder version under several scenarios, For example: The parties dispute whether a defect in the goods happened under the Seller’s or Buyer’s possession. Again, Seller has an advantage, because if she withheld her knowledge about the defect then she probably also factored her risk in advance.

How does one factor-in the risk of the other party being willing to act punitively? If they look at the data from the Ultimatum game, they should be very cautious about incorrectly describing their product and risking the value of their deposit plus the value of the item should the buyer be unsatisfied.

> 2. The Buyer has an advantage when buying a tradeable commodity (e.g. gold bullions)
> 
> Once Buyer receives the goods 50% of his investment is liquid whereas Seller’s entire investment is locked. It seems that he is in a better position to re-negotiate the deal and extort better terms.

The app doesn't allow renegotiation.

> 3. The two-escrow model will fail when the identity of one party is known to the other, and the other party has non-financial motives to damage the first party (political, romantic, revenge, metal illness, etc.).
> 
> In other words, one of the parties may lock the other's funds for non-financial reasons. The assumption that humans are reasonable isn't always true.

Yes, people can choose to loose money to simply harm others but either party that chooses to do this consistently will go bankrupt (which is the sanity test for any stable system). Also, as a major purpose of Bitmarkets is privacy, why would either party share their personal info? The seller can remain completely anonymous and while the buyer does need to provide a delivery address, this isn't necessarily an identity and would require a seller to actively search for identity info related to the address. It seems like an extremely cost ineffective means of trolling compared to other options and it seems to me that the fact that it bankrupts the trollers should be seen as a strong benefit of this system.

> 4. The two-escrow model is susceptible to out-of-band events.
> 
> One of the parties may get arrested, involved in an accident, lose their private keys, etc. those things happen, and under this model it seems that the other party remains with no remedy.

Yes, but this is a much smaller risk than seller collusion with an escrow agent (or the seller actually being the escrow agent) in three party escrow systems as witnessed by the large number of exchange and market exit scams. With two party escrow, exit scams are impossible.
